src/java/main/org/apache/zookeeper/server/quorum/QuorumPeerConfig.java

```diff
@@ -61,6 +61,7 @@
 
     private static boolean standaloneEnabled = true;
     private static boolean reconfigEnabled = false;
+    private static int warnThreshold = 10000;
```

Comment:
Should this have a default value of -1 instead? I think we'd like this to be an opt in feature and by default the logging should be disabled.